tests/Integration/ServiceCompilerTest.php

```diff
@@ -6,16 +6,11 @@
 
 class ServiceCompilerTest extends KernelTestCase
 {
-//    protected function setUp(): void
-//    {
-//        self::bootKernel(['environment' => 'test']);
-//    }
-
     /** @test */
     public function symfonyServiceDefinitionsMustCompile(): void
     {
-        // when this test passed, it means that the kernel could be loaded and there are no compiling errors in the
-        // symfony service definitions.
+        // when this test passed, it means that the kernel could be loaded (boostrap.php) and there are no
```

Comment:
The kernel is not "loaded" (do you mean "booted"?) in the `boostrap.php`, it's only bootstrapped ;)